### PR TITLE
PRIVATE: Use the best pixel format specified by the backend

### PR DESCRIPTION
**Pull request by ccawley2011**

### CODE REVIEW

**Comment by neuromancer** [comment]
Is this ready for review? It looks cool!

**Comment by ccawley2011** [comment]
Sure, go ahead.

File: engines/private/private.cpp [hunk 1]
```diff
@@ -964,7 +964,7 @@ void PrivateEngine::drawScreen() {
 		const Graphics::Surface *frame = _videoDecoder->decodeNextFrame();
 		Graphics::Surface *cframe = frame->convertTo(_pixelFormat, _videoDecoder->getPalette());
 		Common::Point center((_screenW - _videoDecoder->getWidth())/2, (_screenH - _videoDecoder->getHeight())/2);
-		surface->transBlitFrom(*cframe, center);
+		surface->blitFrom(*cframe, center);
```
Comment (neuromancer) [inline engines/private/private.cpp:967]: Why this was changed?

Comment (ccawley2011) [reply engines/private/private.cpp:967]: `transBlitFrom` uses a default transparency colour of 0, which causes graphical errors when using a pixel format without any alpha bits.

Comment (neuromancer) [reply engines/private/private.cpp:1072]: Right, this is unnecessary in during videos.

**Review by neuromancer — APPROVED** [review]

**Pull request merged**